frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -129,6 +128,14 @@ action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-fail=Sli
 action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.of-current=Branch <b>{0}</b> has been slid out but not deleted as it is currently checked out
 action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.with-delete=Branch <b>{0}</b> has been slid out and deleted
 action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.without-delete=Branch <b>{0}</b> has been slid out but not deleted
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.title=Git Machete Branch Deletion Suggestion
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.ok-text=_Delete local branches
```

Comment:
Hmm this is confusing... does it affects the given branch, or all subsequently slid-out branches?...

Reply:
AFAICS it only affects the one given branch, unless `Don't ask again` is selected... still, I'd rather:
* use singular (`Delete/Keep Branch`) and not plural on the buttons
* change `Don't ask again` to sth like `Remember my choice, don't ask again` for clarity

Reply:
Oh ok... so selecting `Don't ask` in this dialog doesn't affect the corresponding git config, right? :facepalm:

Reply:
Hmmm TBH I'd rather not even expose the information about the config to the users... just let's make it possible to set it all up directly from the dialog :thinking:

Reply:
I think unless we have a configuration tab (where settings like this could be easily changed) we SHOULD expose an information about how user can change it.

Reply:
Yeah, okay, you're right, let's have it exposed